### PR TITLE
Feature/negotiation mongo

### PR DESCRIPTION
**Pull request by marest94**

### CODE REVIEW

**Review by IgorBalog-Eng — COMMENTED** [review]
Review

File: connector/src/main/java/it/eng/connector/ApplicationConnector.java
Comment (marest94) [reply connector/src/main/java/it/eng/connector/ApplicationConnector.java:10]: You're right, we can narrow it to .repository, since we are following that pattern in the project structure.

File: connector/src/main/java/it/eng/connector/controller/HelloWorldController.java
Comment (marest94) [reply connector/src/main/java/it/eng/connector/controller/HelloWorldController.java:16]: This will be deleted in the next PR, which will cover deleting all leftovers from the first approach.

File: connector/src/main/java/it/eng/connector/model/User.java
Comment (marest94) [reply connector/src/main/java/it/eng/connector/model/User.java:3]: In fact, this is the code that you wrote, so I left it as is.
When I tried to delete the @AllArgsConstructor, @Builder is not feasible, since Lombok @Builder cannot automatically assume how to handle the object's construction without  an appropriate constructor, so I assume that is the reason why you left it there. So it's either using both of them, or write custom Builder.

And it's not present in other model classes, since in other we implemented the custom builder, and we're not using Lombok's one.

File: negotiation/src/main/java/it/eng/negotiation/service/ContractNegotiationService.java
Comment (marest94) [reply negotiation/src/main/java/it/eng/negotiation/service/ContractNegotiationService.java:71]: Yup, I missed to delete that one.

File: connector/src/main/java/it/eng/connector/controller/HelloWorldController.java
Comment (IgorBalog-Eng) [reply connector/src/main/java/it/eng/connector/controller/HelloWorldController.java:16]: oki

**Review by IgorBalog-Eng — APPROVED** [review]
Once pending issues are resolved - you have approval

**Pull request merged**
